File: API/Models/Produto.cs
namespace API.Models
{
    public class Produto
    {
        public int id { get; set; }
        public string NomeProduto { get; set; }
        public string DescricaoProduto { get; set; }
        public double PrecoProduto { get; set; }
        public int DescontoProduto { get; set; }
        public Farmacia Farmacia { get; set; }

    }
}

File: APP/src/app/models/produto.ts
import { Farmacia } from './farmacia';
import { FormaProduto } from './formaProduto';
export interface Produto {

    id?: number;
    nomeProduto: string;
    descricaoProduto: string;
    precoProduto: number;
    descontoProduto: number;
    farmacia?: Farmacia;
    farmaciaId: number;
    formaProdutoId?: number;
    formaProduto?: FormaProduto;

}
